src/playback/index.jsx

```diff
@@ -0,0 +1,98 @@
+import React from 'react';
+import { withStyles } from '@material-ui/core';
+import Grid from '@material-ui/core/Grid';
+import { selectFrom } from '../vendor/vectors';
+import { BROWSERS, ENCODINGS, PLATFORMS, SIZES, TESTS } from './config';
+import { withNavigation } from '../vendor/utils/navigation';
+import Picker from '../vendor/utils/navigation/Picker';
+import DashboardPage from '../components/DashboardPage';
+import PerfherderGraphContainer from '../containers/PerfherderGraphContainer';
+
+const styles = {
+  chart: {
+    justifyContent: 'center',
+    padding: '1rem',
+  },
+};
+
+class Power extends React.Component {
+  render() {
+    const { classes, navigation, platform, browser, encoding } = this.props;
+    const platformDetails = selectFrom(PLATFORMS)
+      .where({ id: platform })
+      .first();
+    const browserDetails = selectFrom(BROWSERS)
+      .where({ id: browser })
+      .first();
+
+    return (
+      <DashboardPage
+        title="Playback"
+        key={`page_${platform}_${browser}_${encoding}`}>
+        {navigation}
+        <Grid container spacing={24}>
+          {selectFrom(SIZES).map(({ size }) => (
+            <Grid
+              item
+              xs={6}
+              key={`page_${platform}_${browser}_${encoding}_${size}`}
+              className={classes.chart}>
+              <PerfherderGraphContainer
+                title={`Percent dropped frames ${size}`}
+                style={{
+                  'axis.y.format': '{{.|percent}}',
+                  'axis.y.max.max': 1,
+                }}
```

Comment:
The `style` is using [the chart schema](https://github.com/mozilla-frontend-infra/firefox-health-dashboard/blob/master/src/vendor/chartJs/chartSchema.md), and allows me to be specific about the styling.  In this case the Perfherder data is a number between 0.0 and 1.0, so I want my y.max to not go above 1.  Plus I know this number represents a percent: so I format it this way.

This could be simplified even more, by ensuring this domain knowledge comes out of Perfherder, and the chart library knows how to format the axis; but that is too much code for now.